_layouts/default.html

```diff
@@ -12,14 +12,20 @@
     <script src="{{site.baseurl}}/bower_components/vega/vega.min.js"></script>
     <script src="{{site.baseurl}}/vega-lite.min.js"></script>
     <script src="{{site.baseurl}}/bower_components/vega-embed/vega-embed.js" charset="utf-8"></script>
+    <script src="{{site.baseurl}}/bower_components/highlightjs/highlight.pack.js" charset="utf-8"></script>
+
+    <script>var BASEURL = "{{site.baseurl}}";</script>
+    <script src="{{site.baseurl}}/site/static/docs.js" charset="utf-8"></script>
 
     <title> {{site.title}}: {{page.title}} </title>
     <link rel=stylesheet href="{{site.baseurl}}/site/static/main.css">
     <link rel=stylesheet href="{{site.baseurl}}/site/static/syntax.css">
```

Comment:
So if we use highlight, we don't need this any more, right? Doesn't it share the same classes.